test/parallel/test-child-process-execfile.js

```diff
@@ -7,6 +7,7 @@ const { getSystemErrorName } = require('util');
 const fixtures = require('../common/fixtures');
 
 const fixture = fixtures.path('exit.js');
+const execOpts = { encoding: 'utf8', shell: process.env.SHELL };
```

Comment:
`process.env.SHELL` is likely to not be set on Windows so this will end up being undefined.

Reply:
You are correct mr @richardlau, i tested it on another pc running windows and `process.env.SHELL` was undefined.
Thanks for feedback I appreciate it.